### PR TITLE
changelog for the entity cache invalidation

### PR DESCRIPTION
**Pull request by Geal**
The invalidation docs were merged directly to main, and we were waiting for them to make a changelog entry, so it did not end up directly in the 1.55.0. Once this is merged, I'll update the changelog on the releases page

---

**Checklist**

Complete the checklist (and note appropriate exceptions) before the PR is marked ready-for-review.

- [ ] Changes are compatible[^1]
- [ ] Documentation[^2] completed
- [ ] Performance impact assessed and acceptable
- Tests added and passing[^3]
    - [ ] Unit Tests
    - [ ] Integration Tests
    - [ ] Manual Tests

**Exceptions**

*Note any exceptions here*

**Notes**

[^1]: It may be appropriate to bring upcoming changes to the attention of other (impacted) groups. Please endeavour to do this before seeking PR approval. The mechanism for doing this will vary considerably, so use your judgement as to how and when to do this.
[^2]: Configuration is an important part of many changes. Where applicable please try to document configuration examples.
[^3]: Tick whichever testing boxes are applicable. If you are adding Manual Tests, please document the manual testing (extensively) in the Exceptions.

### CODE REVIEW

**Review by chandrikas — APPROVED** [review]

**Review by abernix — APPROVED** [review]
I can't decide if it's better for this to go into `main` or `dev`; I think the answer is perhaps both branches.  Getting it into `dev` first though is good enough.  LGTM.

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,41 @@ This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.
...
+
+> ⚠️ This is a preview for an [Enterprise feature](https://www.apollographql.com/blog/platform/evaluating-apollo-router-understanding-free-and-open-vs-commercial-features/) of the Apollo Router. It requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/).  If your organization doesn't currently have an Enterprise plan, you can test out this functionality with a [free Enterprise trial](https://studio.apollographql.com/signup?type=enterprise-trial).
+>
+> As a preview feature, it's subject to our [Preview launch stage](https://www.apollographql.com/docs/resources/product-launch-stages/#preview) expectations and configuration and performance may change in future releases.
+
+As a follow up to the Entity cache preview that was published in the router 1.46.0 release, we're introducing a new feature that allows you to invalidate cached entries.
+
+This introduces two ways to invalidate cached entries:
+- through an HTTP endpoint exposed by the router
+- via extensions in a subgraph response
```
Comment (abernix) [inline CHANGELOG.md:24]:
```suggestion
- via GraphQL response `extensions` returned from subgraph requests
```

Comment (abernix) [reply CHANGELOG.md:24]: I'll commit this so tests re-run. ;)

**Comment by chandrikas** [comment]
@Geal - shall we merge this?

**Comment by Geal** [comment]
Yes, we just have to fix the tonic vuln thing first so CI passes

**Pull request merged**